microsoft-edge/web-platform/release-notes/136.md

```diff
@@ -182,48 +198,64 @@ See [Fluent Scrollbars Visual Spec](https://docs.google.com/document/d/1haDpb1QI
 <!-- ---------- -->
 ###### AudioContext `interrupted` state
 
-Currently, the Web Audio API doesn't have a mechanism for the browser to interrupt playback for scenarios such as exclusive audio access (VoIP) or when a laptop lid is closed. This change introduces the `interrupted` state to the `AudioContextState` interface. This new state allows the browser to pause playback in the above scenarios and enables web applications to respond appropriately.
+The `AudioContextState` interface now has an `interrupted` state.  The `interrupted` state allows the browser to pause playback in scenarios such as when using exclusive audio access (VoIP) or when a laptop lid is closed, and enables the web app to respond appropriately.
 
 See [AudioContext](https://developer.mozilla.org/docs/Web/API/AudioContext) at MDN.
 
 
 <!-- ---------- -->
-###### Blob URL Partitioning: Fetching/Navigation
+###### Blob URL Partitioning: Fetching/Navigation<!-- todo: section lacks "fetching", try to avoid / and spell out -->
+
+<!-- behavior 1 -->
+Blob URL access is now partitioned by storage key.  Storage keys are top-level sites, frame origins, and the `has-cross-site-ancestor` boolean.  Top-level navigations remain partitioned only by frame origin.
+
+Partitioning Blob URL access by storage key is similar to what's currently implemented by both Firefox and Safari.  This aligns Blob URL usage with the partitioning scheme that's used by other storage APIs as part of Storage Partitioning.
 
-Blob URL access is now partitioned by storage key. Storage keys are top-level sites, frame origins, and the has-cross-site-ancestor boolean. Top-level navigations remain partitioned only by frame origin. This behavior is similar to what's currently implemented by both Firefox and Safari, and aligns Blob URL usage with the partitioning scheme used by other storage APIs as part of Storage Partitioning.
+<!-- behavior 2 -->
+Along with that change, Microsoft Edge now enforces `noopener` on renderer-initiated top-level navigations to Blob URLs, where the corresponding site is cross-site to the top-level site that's performing the navigation.  This aligns with similar behavior in Safari and with the corresponding specifications.
 
-In addition, Edge now enforce noopener on renderer-initiated top-level navigations to Blob URLs where the corresponding site is cross-site to the top-level site performing the navigation. This aligns with similar behavior in Safari and with the corresponding specifications.
+This change can be temporarily reverted by setting the `PartitionedBlobUrlUsage` policy.<!-- todo: applies to behavior 1 & 2? -->  If this policy is set to `Enabled` or is not set, Blob URLs are partitioned.<!-- todo: applies to behavior 1 & 2? -->  If this policy is set to `Disabled`, Blob URLs are not partitioned.
```

Comment:
section describes two distinct behavior changes, but then says "this change" - explicitly clarify w/ hints, whether "this change" means both aspects.